app/Console/Commands/UiTPAS/UiTPASIntegration.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands\UiTPAS;
+
+use App\Auth0\Auth0Tenant;
+use App\Domain\Contacts\Contact;
+use App\Domain\Contacts\ContactType;
+use App\Domain\Integrations\IntegrationStatus;
+use App\Domain\Integrations\Website;
+use InvalidArgumentException;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+
+final class UiTPASIntegration
```

Comment:
I would change this DTO object in the following ways:
- In the constructor expect all the specifiek fields (id, email, lastname) as separate arguments, and give them their own properties.
- Have a method "fromArray" to accepts the current array and splits up all the fields.

This would have several benefits:
- Would make this class more reusable for other situations / cleaner to mock
- Gives you type hinting for all the separate fields
- Makes the code more readable because you only have the indexes on one place and don't have to remember "index 2 is the email", etc...

Reply:
The goal of this class is to wrap the row of the CSV file, if I would change it to accept typed parameters there is no longer a need for this class. I could rename it to `UiTPASIntegrationFromCSV` or something like that.